modules/kmm-configuring-out-of-tree-module-images.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="kmm-configuring-out-of-tree-module-images_{context}"]
+= Configuring out-of-tree module images
+
+If you are using out-of-tree (OOT) modules, you can use the image configuration settings in OpenShift's core image configuration (`image.config.openshift.io/cluster`) to configure the modules instead of configuring each module configuration. 
+
```

Comment:
@ybettan Do we need something here explaining why OOT modules need to be configured with insecure registries? One sentence will do, I think.

Reply:
I think the idea is to say that they can be configured with insecure registries, rather than that they need to be. 
Especially because OOT modules can be written by anyone that needs to do so. 
Sometimes I presume, these people do not have access to secure registries or want to use their own insecure registries. Similar to, I guess, to any other reason for using those. That is why it is all set in the general image registry settings of the cluster as such. (I think this is mentioned as a link in the doc too, so it should be OK)